docs/advanced-customization/config/email-digest-frequency.md

```diff
@@ -1,17 +0,0 @@
----
-description: Modify email digest frequency.
```

Comment:
We've removed the Email Digest Frequency Section and added it to the Emails section